build/packages.targets

```diff
@@ -89,7 +89,7 @@
 
     <!-- Test and utility packages -->
     <ItemGroup>
-        <PackageReference Update="FluentAssertions" Version="5.4.1" />
+        <PackageReference Update="FluentAssertions" Version="6.0.0-alpha0002" />
```

Comment:
Why that specific preview version? There's stable versions higher than this.

Reply:
We chatted offline about the specifics. This version and higher contains NotContainMatch that is used in the tests to evaluate the output console messages.

Reply:
yes, but there's a stable 6.0.0 version: https://www.nuget.org/packages/FluentAssertions/6.0.0

and the latest version (which happens to be stable) is 6.6.0: https://www.nuget.org/packages/FluentAssertions/6.6.0

Reply:
those versions are not mirrored from nuget.org yet. 
There must be some ingestion process.

Reply:
6.0.0 was published to nuget.org 8 months ago.  We should contact dnceng.

Reply:
yep, that's the plan.

Reply:
@zivkan:
>edit: if NotContainMatch was removed from the stable versions, then we shouldn't use it.

NotContainMatch is still present in all versions after it was introduced. https://github.com/fluentassertions/fluentassertions/blob/9e03620a904861a7c5aa66de00283a36155cc8e4/Src/FluentAssertions/Collections/StringCollectionAssertions.cs#L319

Reply:
6.6.0 is now available.

Reply:
Updated the PR to use 6.6.0. Thanks!!